load-gen/src/main/java/io/smartcat/data/loader/api/RateGenerator.java

```diff
@@ -0,0 +1,6 @@
+package io.smartcat.data.loader.api;
+
+public interface RateGenerator {
+
+    long getRate(long prevInstant, long currentInstant);
```

Comment:
If we have two variations, how should LoadGenerator know which one to use?
Second, if we have just current time, implementation needs to change in a way that we have timer with some uniform ticks in order to know how to calculate the rate.